index.html

```diff
@@ -1,19 +1,129 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <title>Karma</title>
-    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
-    <link rel="stylesheet" href="css/normalize.css">
-    <link rel="stylesheet" href="css/style.css">
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <title>Karma</title>
+  <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
+  <link rel="stylesheet" href="css/normalize.css">
+  <link rel="stylesheet" href="css/style.css">
+  <link rel="shortcut icon" type="image/x-icon" href="favicon.ico">
 </head>
+
 <body>
 
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
+  <!-- Add your HTML markup here -->
+  <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+  <!-- All the images you need are in the 'img' folder -->
+  <header class="header">
+
+    <a href="#">
+      <img src="img/karma-logo.svg" alt="Logo of karma" class="header__logo" />
+    </a>
+
+    <nav>
+      <ul class="navigation__list">
+        <li class="navigation__item">
+          Meet Karma
+        </li>
+        <li class="navigation__item">
+          <a class="navigation__link" href="#">How it works</a>
+        </li>
+        <li class="navigation__item">
+          <a class="navigation__link" href="/store.html">Store</a>
+        </li>
+        <li class="navigation__item">
+          <a class="navigation__link" href="#">Blog</a>
+        </li>
+        <li class="navigation__item">
+          <a class="navigation__link" href="#">Help</a>
+        </li>
+        <li class="navigation__item">
+          <a class="navigation__link" href="#">Login</a>
+        </li>
+      </ul>
+    </nav>
+  </header>
+  <main class="content">
+    <div class="main">
+
+
+      <div class="hero">
```

Comment:
you can also use semantic html instead of div, such as <section> which you use them in some parts of your code 👍